modules/vstudio/_preload.lua

```diff
@@ -197,7 +197,7 @@
 	}
 
 	p.api.register {
-		name = "enablemodules",
+	name = "enablemodules",
```

Comment:
This style change seems wrong.

Reply:
Let me check this again, it is completely unintentional, I merged in the web interface and it introduced differences that I tried to fix locally after, won't be a minute

Reply:
Fixed now, thanks